### PR TITLE
Fixes typos and wrong filepath

### PR DESCRIPTION
**Pull request by marquicodes**
Very nice project! It came on time as I was looking how to use Svelte with Fastify. 👍🏼

I found a few typos and a wrong file path.

Great job.

### CODE REVIEW

**Review by delvedor — CHANGES_REQUESTED** [review]
Thanks! Can you rebase on master?

File: test/helper.js [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
 
 // We need to load the env configuration
-dotenv.config({ path: join(import.meta.url, '..', '.env') })
+process.env.NODE_ENV = 'development'
+process.env.ELASTIC_ADDRESS = 'http://localhost:9200'
+process.env.ELASTIC_API_KEY = '<api-key>'
+process.env.GITHUB_APP_ID = '<app-id>'
+process.env.GITHUB_APP_SECRET = '<app-secret>'
+process.env.COOKIE_SECRET = '<cookie-secret>'
+process.env.ALLOWED_USERS = 'foo@bar.com'
```
Comment (delvedor) [inline test/helper.js:13]: This change is already in `main`, not sure why it's present here as well. 🤔

**Review by delvedor — APPROVED** [review]
My bad! I meant to merge the main branch in yours!
These changes are ok, thanks for contributing!

**Comment by marquicodes** [comment]
Sorry for misunderstanding. Thank you for creating this very useful example.

**Pull request merged**